.gitattributes

```diff
@@ -1 +1,7 @@
-app/javascript/oldjs/locale/*.json -text -diff
+# To use gron for human readable json differences:
+#   1) brew/apt install gron
+#   2) Add this to ~/.gitconfig:
+#     [diff "gron"]
+#          textconv=gron
+# Now, git log -p, show, and friends will use gron for json differences.
+app/javascript/oldjs/locale/*.json diff=gron
```

Comment:
Based on @Fryguy's suggestion, I tried this without gron installed and `.gitconfig` not configured to use gron, this `diff=gron` on line 7 basically does nothing as `diff` reverts to the default differ.